docs-samples/data-science/semantic-link-samples/great_expectations_tutorial.ipynb

```diff
@@ -0,0 +1,2642 @@
+        "* Open your notebook. You have two options:\n",
+        "    * [Import this notebook into your workspace](https://learn.microsoft.com/en-us/fabric/data-engineering/how-to-use-notebook#import-existing-notebooks). You can import from the Data Science homepage.\n",
+        "    * Alternatively, you can create [a new notebook](https://learn.microsoft.com/fabric/data-engineering/how-to-use-notebook#create-notebooks) to copy/paste code into cells."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "f003ff81-c3e0-4f14-9336-6bd21629a860",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "## Set up the notebook\n",
+        "\n",
+        "In this section, you set up a notebook environment with the necessary modules and data.\n",
+        "\n",
+        "1. Install `SemPy` from PyPI using the `%pip` in-line installation capability within the notebook."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "id": "450c8f7b-6896-4349-8101-014057015aa8",
+      "metadata": {
+        "jupyter": {
+          "outputs_hidden": true,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "tags": [
+          "fabric_skip_execution"
+        ]
+      },
+      "outputs": [
+        {
+          "data": {
+            "application/vnd.livy.statement-meta+json": {
+              "execution_finish_time": "2023-10-06T15:55:05.7366091Z",
+              "execution_start_time": "2023-10-06T15:54:54.5912694Z",
+              "livy_statement_state": "available",
+              "parent_msg_id": "08fd856e-a076-485a-b3c1-31ae9afac96c",
+              "queued_time": "2023-10-06T15:50:38.4731581Z",
+              "session_id": "efb436d1-09b2-43ab-a318-5b84c484989f",
+              "session_start_time": null,
+              "spark_jobs": {
+                "jobs": [],
+                "limit": 20,
+                "numbers": {
+                  "FAILED": 0,
+                  "RUNNING": 0,
+                  "SUCCEEDED": 0,
+                  "UNKNOWN": 0
+                },
+                "rule": "ALL_DESC"
+              },
+              "spark_pool": null,
+              "state": "finished",
+              "statement_id": 10
+            },
+            "text/plain": [
+              "StatementMeta(, efb436d1-09b2-43ab-a318-5b84c484989f, 10, Finished, Available)"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "data": {},
+          "execution_count": 1,
+          "metadata": {},
+          "output_type": "execute_result"
+        },
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "Collecting semantic-link\n",
```

Comment:
can we hide the pip install text from the output?

Reply:
sure